accordion/index.html

```diff
@@ -0,0 +1,113 @@
+            float: right;
+            margin-left: 5px;
+        }
+
+        .active:after {
+            content: "\25BC";
+        }
+
+        .section {
+            padding: 0 18px;
+            background-color: #f8f8f8;
+            max-height: 0;
+            overflow: hidden;
+            transition: max-height 0.2s ease-out;
+        }
+    </style>
+</head>
+
+<body>
+
+    <h2>Accordion Example</h2>
+    <button class="accordion">Section 1</button>
+    <div class="section">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec dictum ipsum, sed maximus massa.
+            Curabitur eleifend
+            lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+            Pellentesque
+            facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+            posuere luctus
+            imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+            orci in,
+            condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+            lectus ac
+            vehicula.</p>
+    </div>
+
+    <button class="accordion">Section 2</button>
+    <div class="section">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec dictum ipsum, sed maximus massa.
+            Curabitur eleifend
+            lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+            Pellentesque
+            facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+            posuere luctus
+            imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+            orci in,
+            condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+            lectus ac
+            vehicula.</p>
+    </div>
+
+    <button class="accordion">Section 3</button>
+    <div class="section">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec dictum ipsum, sed maximus massa.
+            Curabitur eleifend
+            lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+            Pellentesque
+            facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+            posuere luctus
+            imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+            orci in,
+            condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+            lectus ac
+            vehicula.</p>
+    </div>
+
+    <script>
+        var accordion = document.getElementsByClassName("accordion");
```

Comment:
Also, instead of playing with the _height_ - which is an expensive operation for the browser (DOM Reflow / redraw) 
[Here](https://medium.com/darrja-%E0%A4%A6%E0%A4%B0%E0%A5%8D%E0%A4%9C%E0%A4%BE/what-the-heck-is-repaint-and-reflow-in-the-browser-b2d0fb980c08)

What we can do is, we can play with _hidden_ property. Make a class _hide_ and _toggle_ that.

Reply:
Thanks for the review. I have updated the code. Please review.